_src/static/sass/utilities/_variables.scss

```diff
@@ -178,31 +186,32 @@ $palettes: (
 @include font-face('Open Sans', '../fonts/OpenSans/OpenSans-Bold-webfont', 700, $file-formats: eot woff ttf svg);
 @include font-face('Open Sans', '../fonts/OpenSans/OpenSans-BoldItalic-webfont', 700, italic, $file-formats: eot woff ttf svg);
 
-// font families
+// typography: config - stacks
 $font-family-sans-serif: 'Open Sans','Helvetica Neue', Helvetica, Arial, sans-serif;
 $font-family-monospace: 'Bitstream Vera Sans Mono', Consolas, Courier, monospace;
 
-// scale roots
-$font-scale-small: 14;
-$font-scale-medium: 16;
-$font-scale-large: 16;
-$font-scale-base: $font-scale-medium;
+// typography: config - scale
+$modular-scale-ratio: $perfect-fourth;
 
-$em-base: $font-scale-medium; // deliberately sets bourbon-based em-base (http://bourbon.io/docs/#em-base)
+$font-scale-small: 16;
+$font-scale-medium: 18;
+$font-scale-large: 18;
+$font-scale-base: $font-scale-medium;
 
+// typography: sizes
 $font-sizes: (
     base:           $font-scale-base,
-    mid-small:      ceil(golden-ratio($font-scale-base, -1)),
-    small:          ceil(golden-ratio($font-scale-base, -2)),
-    x-small:        ceil(golden-ratio($font-scale-base, -3)),
-    xx-small:       ceil(golden-ratio($font-scale-base, -4)),
-    mid-large:      ceil(golden-ratio($font-scale-base, 1)),
-    large:          ceil(golden-ratio($font-scale-base, 2)),
-    x-large:        ceil(golden-ratio($font-scale-base, 3)),
-    xx-large:       ceil(golden-ratio($font-scale-base, 4)),
+    mid-small:      modular-scale(-1, $font-scale-base),
```

Comment:
NOTE: switching to this method since ``golden-ratio`` is deprecated in the most recent version of Bourbon.